psse/psse-converter/src/main/java/com/powsybl/psse/converter/LoadConverter.java

```diff
@@ -34,18 +34,26 @@ void create() {
         VoltageLevel voltageLevel = getNetwork()
             .getVoltageLevel(getContainersMapping().getVoltageLevelId(psseLoad.getI()));
 
-        // Only constant power is considered at the moment
-        // S = VI*, S = YVV*
-        // .setIp(psseLoad.getIp())
-        // .setIq(psseLoad.getIq())
-        // .setYp(psseLoad.getYp())
-        // .setYq(psseLoad.getYq())
+        double p0 = psseLoad.getPl() + psseLoad.getIp() + psseLoad.getYp();
+        double q0 = psseLoad.getQl() + psseLoad.getIq() + psseLoad.getYq();
 
         LoadAdder adder = voltageLevel.newLoad()
             .setId(getLoadId(busId))
             .setConnectableBus(busId)
-            .setP0(psseLoad.getPl())
-            .setQ0(psseLoad.getQl());
+            .setP0(p0)
+            .setQ0(q0);
+
+        boolean constantPower = psseLoad.getIp() == 0 && psseLoad.getYp() == 0 && psseLoad.getIq() == 0 && psseLoad.getYq() == 0;
+        if (!constantPower) {
+            adder.newZipModel()
+                    .setC0p(psseLoad.getPl() / p0)
```

Comment:
`p0` or `q0` may be zero:
ZIP model factors are calculated if any of the `p`, `q` values of I/Y parts is non-zero.
A user may define a I/Y model with `p != 0` but all `q = 0`, then `q0` would be zero.
Maybe protect the potential division by zero (it happens also in CGMES import, using the coefficients).

Reply:
Done